src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/SyntaxTreeExtensions.cs

```diff
@@ -417,7 +417,12 @@ private static bool AtEndOfIncompleteStringOrCharLiteral(SyntaxToken token, int
                 token = token.GetPreviousToken(includeSkipped: true, includeDirectives: true);
             }
 
-            if (token.Kind() is SyntaxKind.StringLiteralToken or SyntaxKind.SingleLineRawStringLiteralToken or SyntaxKind.MultiLineRawStringLiteralToken)
+            if (token.IsKind(SyntaxKind.StringLiteralToken,
+                             SyntaxKind.SingleLineRawStringLiteralToken,
+                             SyntaxKind.MultiLineRawStringLiteralToken,
```

Comment:
I see these already existed before your change. But I'm confused with it when reading this comment:

https://github.com/dotnet/roslyn/blob/c8ebc8682889b395fcb84c85bf4ff54577377d26/src/Analyzers/CSharp/CodeFixes/ConvertNamespace/ConvertNamespaceTransform.cs#L126-L129

Reply:
Yes, interesting disconnect there. The comment isn't wrong, in that raw strings can be _potentially_ dedented, but definitely outside the scope of this PR.